pbm/storage/s3/s3.go

```diff
@@ -88,6 +125,42 @@ func (c *Conf) Cast() error {
 	return nil
 }
 
+// SDKLogLevel returns AWS SDK log level value from comma-separated
+// SDKDebugLogLevel values string. If the string is empty, returns aws.LogOff.
+//
+// If the string is incorrect formatted, prints warnings to the io.Writer.
+// Passing nil as the io.Writer will discard any warnings.
+func SDKLogLevel(levels string, out io.Writer) aws.LogLevelType {
+	if levels == "" {
+		return aws.LogOff
+	}
+
+	if out == nil {
+		out = ioutil.Discard
+	}
+
+	var logLevel aws.LogLevelType
+	ss := make(map[string]struct{})
+
+	for _, lvl := range strings.Split(levels, ",") {
+		l := SDKDebugLogLevel(lvl).SDKLogLevel()
+		if l == 0 {
+			fmt.Fprintf(out, "WARN: S3 client debug log level: unsupported %q\n", lvl)
+			continue
+		}
+
+		if _, ok := ss[lvl]; ok {
+			fmt.Fprintf(out, "WARN: S3 client debug log level: duplicated %q\n", lvl)
```

Comment:
s/WARN/Warning
In other places in CLI we are using the`"Error:.."` format. `"Warning: ..."` would be more consistent